lib/generators/arqo/named_base.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+require 'rails/generators/named_base'
+
+module ARQO
+  module Generators
+    class NamedBase < Rails::Generators::NamedBase
+      argument :attributes, type: :array, default: [], banner: 'associated_to[:class_name]'
```

Comment:
maybe `associated_relation` is better than `associated_to` ? not sure what the rest thinks though..